src/container/srv_target.c

```diff
@@ -78,6 +78,10 @@ cont_aggregate_runnable(struct ds_cont_child *cont)
 	/* snapshot list isn't fetched yet */
 	if (cont->sc_aggregation_max == 0) {
 		D_DEBUG(DB_EPC, "No aggregation before snapshots fetched\n");
+		/* fetch snapshot list */
+		if (dss_get_module_info()->dmi_tgt_id == 0)
+			ds_cont_tgt_snapshots_refresh(cont->sc_pool->spc_uuid,
+						      cont->sc_uuid);
```

Comment:
Can there be more than one cont_snapshots_refresh_ult ULTs running at the same time? We're creating anonymous ULTs again (but not a fault of this PR).